cmd/orb.go

```diff
@@ -615,5 +617,12 @@ func orbInfo(opts orbOptions) error {
 	fmt.Printf("Projects: %d\n", info.Orb.Statistics.Last30DaysProjectCount)
 	fmt.Printf("Orgs: %d\n", info.Orb.Statistics.Last30DaysOrganizationCount)
 
+	orbVersionSplit := strings.Split(ref, "@")
+	orbRef := orbVersionSplit[0]
+	fmt.Printf(`
+Learn more about this orb online in the CircleCI Orb Registry:
+https://circleci.com/orbs/registry/orb/%s
+`, orbRef)
```

Comment:
Can we base this on the value of host?

Reply:
@ndintenfass Yes, but I want to rebase that after #247 is merged